src/python/doc/bottleneck_distance_sum.inc

```diff
@@ -8,7 +8,7 @@
    |                                                   | diagonal points) such that any couple of matched points are at       |                                                                 |
    |      Bottleneck distance is the length of         | distance at most b, where the distance between points is the sup     | :License: MIT (`GPL v3 </licensing/>`_)                         |
    |      the longest edge                             | norm in :math:`\mathbb{R}^2`.                                        |                                                                 |
-   |                                                   |                                                                      | :Requires: `CGAL <installation.html#cgal>`_ :math:`\geq` 4.11.0 |
+   |                                                   |                                                                      | :Requires: `CGAL <installation.html#cgal>`_ :math:`\geq` 5.1.0  |
```

Comment:
I don't know if having the version number repeated everywhere is that useful (it isn't horrible though). A possibility would be to specify it only when it isn't the "default" one.